plugins/inventory/linode.py

```diff
@@ -229,8 +246,8 @@ def parse(self, inventory, loader, path, cache=True):
         self._get_instances_inventory()
 
         strict = self.get_option('strict')
-        regions, types = self._get_query_options(config_data)
-        self._filter_by_config(regions, types)
+        regions, types, tags = self._get_query_options(config_data)
```

Comment:
This can probably be simplified to
```suggestion
        regions = self.get_option('regions')
        types = self.get_option('types')
        tags = self.get_option('tags')
```
But that would be something for another PR.